libweb3jsonrpc/Eth.cpp

```diff
@@ -57,26 +57,12 @@ string Eth::eth_coinbase()
 
 string Eth::eth_hashrate()
 {
-	try
-	{
-		return toJS(asEthashClient(client())->hashrate());
-	}
-	catch (InvalidSealEngine&)
-	{
-		BOOST_THROW_EXCEPTION(JsonRpcException(Errors::ERROR_RPC_INVALID_PARAMS));
-	}
+    return toJS(getEthash().hashrate());
```

Comment:
This is not quite good, this way we'll miss any exception coming from `hashrate()` or `toJS()` and it will crash... I suggest leaving `try/catch` here with two `catch` clauses - for `InvalidSealEngine` and for `std::exception`